pages/juang961225/style.css

```diff
@@ -0,0 +1,115 @@
+body {
+    display: flex;
```

Comment:
I think it's not necessary to set the property flex for the body because this will compromise any other element that could be added as its direct children.

Reply:
solved